django-stubs/views/generic/list.pyi

```diff
@@ -18,7 +18,7 @@ class MultipleObjectMixin(Generic[T], ContextMixin):
     paginator_class: Type[Paginator] = ...
     page_kwarg: str = ...
     ordering: Sequence[str] = ...
-    def get_queryset(self) -> QuerySet[T]: ...
+    def get_queryset(self) -> Union[QuerySet[T], Iterable[Any]]: ...
```

Comment:
`Iterable[Any]` or `Iterable[T]`?

Reply:
One more thing, we try to use less `Union` return types, where possible.
This will force our users to always check what exactly was returned. Which is not very nice.